src/features/usage-consolidation-accordion/usage-consolidation-content-resource/usage-consolidation-content-resource.test.js

```diff
@@ -0,0 +1,116 @@
+import { render } from '@testing-library/react';
+
+import Harness from '../../../../test/jest/helpers/harness';
+
+import UsageConsolidationContentResource from './usage-consolidation-content-resource';
+import { costPerUseTypes } from '../../../constants';
+
+const costPerUseData = {
+  data: {
+    [costPerUseTypes.RESOURCE_COST_PER_USE]:{
+      attributes: {
+        analysis: {
+            cost: 1,
+            costPerUse: 1,
+            usage: 1
+        },
+        parameters: {
+            currency: '',
+            startMonth: '',
+        },
+        usage:{
+            platforms:[
+                {
+                    counts: [3, 6, 16],
+                    isPublisherPlatform: true,
+                    name: "name",
+                    total: 17
+                }
+            ],
+            totals:{}
+          }
+        },
+      },
+  },
+  isFailed: false,
+  isLoaded: true,
+  isLoading: false,
+};
+
+const renderUsageConsolidationContentResource = (props = {}) => render(
+  <Harness>
+    <UsageConsolidationContentResource
+      costPerUseData={costPerUseData}
+      {...props}
+    />
+  </Harness>
+);
+
+describe("Given UsageConsolidationContentResource", () => {
+
+  it('No DOMelement should be rendered when costPerUseData.data.resourceCostPerUse is null', () => {
```

Comment:
We use unit tests naming convention where `it` clause starts with "should...". And also there's no need to add an empty line here.

```suggestion
describe("Given UsageConsolidationContentResource", () => {
  it('should not render DOMelement when costPerUseData.data.resourceCostPerUse is null', () => {
```

Reply:
Implemented the changes in latest commit